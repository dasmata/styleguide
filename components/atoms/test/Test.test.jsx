import React from 'react';
import { shallow } from 'enzyme';

import Test from './Test';

describe('creating a test test', () => {
  it("snapshot", () => {
    const tree = shallow(<Test a='vsvs' />);
    expect(tree).toMatchSnapshot();
  });
});
