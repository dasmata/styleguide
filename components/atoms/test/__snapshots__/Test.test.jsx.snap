// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creating a test test snapshot 1`] = `ShallowWrapper {}`;
